enhancements/assets-generation/cloud-foundry-application-discovery/README.md

```diff
@@ -229,13 +229,21 @@ be a good place to talk about core concepts and how they relate.
 
 **Carefully think through the security implications for this change**
 
-What are the risks of this proposal and how do we mitigate. Think broadly. How
-will this impact the broader OKD ecosystem? Does this work in a managed services
-environment that has many tenants?
+_What are the risks of this proposal and how do we mitigate. Think broadly. How_
+_will this impact the broader OKD ecosystem? Does this work in a managed services_
+_environment that has many tenants?_
 
-How will security be reviewed and by whom? How will UX be reviewed and by whom?
+_How will security be reviewed and by whom? How will UX be reviewed and by whom?_
 
-Consider including folks that also work outside your immediate sub-project.
+_Consider including folks that also work outside your immediate sub-project._
+
+#### Canonical Manifest Misalignment
+Generated canonical manifests may not fully capture the original CF manifests,
```

Comment:
I´d be more assertive and state that CF provides features that are not directly mappeable to K8S (buildpacks), therefore in such scenarios, the migration to a K8S environment will require additional work before they are deployed.
Although this is going a bit too far in the scope of work here (discovery). But `buildpacks` are a perfect example. WDYT?

Reply:
Also docker secrets. And services, which work differently in CF vs K8s.